src/components/Form/DateControl/DateControl.jsx

```diff
@@ -260,32 +261,39 @@ export default class DateControl extends ValidationElement {
   }
 
   render () {
+    console.log('month:', this.state.month)
```

Comment:
Yeah removed this in a recent commit on the `military-history` branch. Definitely was not meant to stick around.